documentation/starter-kits/GOVERNANCE/GOVERNANCE.md

```diff
@@ -0,0 +1,117 @@
+
+The project follows a fairly liberal contribution model where people and/or organizations who do the most work will have the most influence on project direction. Technical decision making will primarily be made through a "[consensus-seeking](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making)" approach. 
+
+## Roles
+
+### User
+
+Anyone who has downloaded, deployed, or operated [INSERT PROJECT NAME] to meet a specific objective. [INSERT PROJECT NAME] was primarily designed for [INSERT DESCRIPTION OF PROJECT PURPOSE], but let us know if you've found other uses for it.  
+
+### Contributor
+
+Contributors include anyone that provides input to the project. This includes code, issues, documentation, graphics, designs, or anything else that tangibly improves the project. We encourage you to start contributing right away by joining our [Discussions]([INSERT LINK TO DISCUSSION BOARD]) or submitting an [Issue]([INSERT LINK TO ISSUE TRACKING SYSTEM]). 
+
+### Triager
+
+Subset of contributors who have demonstrated a strong familiarity with [INSERT PROJECT NAME] and are regularly contributing to the project via issue creation, commenting, discussions, etc. Triagers are given the "Triage" GitHub role and are added to corresponding GitHub team. These team members have the ability to do the following:
+
+    - Label issues and pull requests
+    - Comment, close, and reopen issues and pull requests
+
+[List of current Triagers]([INSERT LINK TO GITHUB TEAM ASSOCIATED WITH TRIAGERS])
+ 
+### Collaborator
+
+Subset of contributors who have been given write access to one or more of the [INSERT PROJECT NAME] repositories. Both contributors and collaborators can propose changes to the project via pull requests, but only collaborators can formally review and approve (merge) these requests. Any contributor who has made a non-trivial contribution should be on-boarded as a collaborator in a timely manner. 
+
+If you are planning on making a substantial contribution to the project or feel as though you should be given write access to a repository, please send a request to [INSERT LINK TO EMAIL/GITHUB USER HANDLE]
+
+[List of current collaborators]([INSERT LINK TO GITHUB TEAM ASSOCIATED WITH COLLABORATORS])
+
+## Technical Steering Committee
+
+A subset of the collaborators forms the Technical Steering Committee (TSC). The TSC has authority over the following aspects of this project:
+
+- Technical direction and guidance
+- Committee governance and process 
+- Contribution policy
+- Conduct guidelines
+- Maintaining the list of collaborators
+
+### TSC Committee Members
+- [INSERT MEMBER NAME] ([username1]([INSERT LINK TO USERNAME]), [INSERT ORG ASSOCIATION]
+- [INSERT MEMBER NAME] ([username1]([INSERT LINK TO USERNAME]), [INSERT ORG ASSOCIATION]
+
+
+<details>
+
+<summary>Emeriti</summary>
+
+### TSC Emeriti
+- [INSERT MEMBER NAME] ([username1]([INSERT LINK TO USERNAME]), [INSERT ORG ASSOCIATION]
+
+</details>
+ 
+### Scope

```

Comment:
We don't typically list the names of individual repos for other repo metadata files like `CONTRIBUTING.md`, `CODE_OF_CONDUCT.md`, `LICENSE` etc., due to the overhead of maintaining repository names for multi-repo projects. 

Is there a benefit to listing the repos applicable to the governance policy, as opposed to making a statement such as "the repositories containing this `GOVERNANCE.md` are subject to the policies described within it" or something to that effect?